### ISSUE TITLE
The description is not set during install

### ISSUE THREAD

**Issue by AdrianLogan** [SharpeRAD/Cake.Services#10]
When specifying a description via the install parameters, the [description](https://github.com/SharpeRAD/Cake.Services/blob/431e8e0cc4ffdcb67435d5f4e96a04306aff2d10/src/Services/Manager/ServiceManager.cs#L998) is set as a separate command after the install. 
The issue is that no service name is supplied as part of this command so it does not execute, just displays the usage help which does not result in error in the build script.

See [MS help](https://technet.microsoft.com/en-us/library/cc742069(v=ws.11).aspx) for details

**Comment by SharpeRAD** [comment]
Fixed in v0.2.9 👍

**Comment by h82258652** [comment]
It occurs again. After installed the service, the description is empty.
Here is my script:
```
Task("Install-Service")
    .Does(()=>
{
    InstallService(new InstallSettings()
    {
        ServiceName = "TestService",
        DisplayName = "My Test Service",
        Description = "My Test Service Description",

        ExecutablePath = @"C:\Projects\TestService\bin\Debug\TestService.exe",
        StartMode = "auto"
    });
});
```
Cake.Services version is 0.3.6
